test/models/comment_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
+   test "comment_on_post" do 
```

Comment:
I'm very much unfinished with the tests. I was going through all the rigmarole of the TDD lessons to try and solidify my TDD mindset. These will progress throughout the day today and tomorrow  as I change my tests, I was just jumping through all those "unnecessary" hoops to try and really understand what was happening.   If I can leave them for today they are kind of a place holder for me to work through  my TDD process.  (which is slow very very slow).  Is it bad form to have placeholders in your pull requests?  I might have inaccurately assumed they would not affect anyone's work.  If I need to delete them please let me know and I will.